features/step_definitions/volunteer_op_steps.rb

```diff
@@ -37,7 +37,7 @@
 end
 
 Then(/^there should be (\d+) doit volunteer ops stored$/) do |count|
-  expect(VolunteerOp.count(source: 'doit')).to eq count.to_i
+  expect(VolunteerOp.count_by_sql("SELECT COUNT(*) FROM volunteer_ops WHERE source = 'doit'")).to eq count.to_i
```

Comment:
I'm kind of surprised about this - seems like it's associated with upgrade to Rails 4.2.7? weird though that a version of count would stop working from one minor version to the next - maybe our old cucumber version was using a much older version of rails AR?